test/tools-test.cpp

```diff
@@ -0,0 +1,196 @@
+#include "gtest/gtest.h"
+
+#include "../src/tools.h"
+#include <magic.h>
+
+std::string directoryPath;
+magic_t magic;
+bool inflateHtmlFlag = false;
+bool isVerbose() { return false; }
+
+TEST(CommonTools, getFileSize)
+{
+  std::string fn = "data/minimal-content/favicon.png";
+
+  system("pwd");
```

Comment:
this system() syscall was probably for debugging?

Reply:
Yes, it was for debug, removed. Excuse me.